tests/system/src/testscriptapp1/java/mymod/myapp1/Util.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2017, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove "2017, "

Reply:
Since this file was copied verbatim from one of the other test apps, the initial `2017,` is correct.